src/Lens.js

```diff
@@ -0,0 +1,36 @@
+const
+  R      = require('ramda'),
+  RF     = require('ramda-fantasy'),
+  PF     = require('./Internal/Profunctor/Class/Profunctor'),
+  Strong = require('./Internal/Profunctor/Class/Strong'),
+
+  Maybe = RF.Maybe;
+  Tuple = RF.Tuple;
+
+
+// lens_ :: (s -> Tuple a (b -> t)) -> Lens s t a b
+const lens_ = R.curry((to, pab) =>
+  PF.dimap(to, t => Tuple.snd(t)(Tuple.fst(t)), Strong.first(pab)));
+
+// lens :: (s -> a) -> (s -> b -> t) -> Lens s t a b
+const lens = R.curry((getter, setter) =>
+  lens_(s => Tuple(getter(s), b => setter(s, b))));
+
+// _1 :: Lens (Tuple a c) (Tuple b c) a b
+const _1 = Strong.first;
+
+// _2 :: Lens (Tuple c a) (Tuple c b) a b
+const _2 = Strong.second;
+
+// atObject :: String -> LensP (Object a) (Maybe a)
```

Comment:
Theses types were following the convention of the [purescript-profunctor-lenses types](https://pursuit.purescript.org/packages/purescript-profunctor-lenses/0.4.0/docs/Data.Lens.Types), to which I presume followed the convention of [Kmett's type aliases](https://hackage.haskell.org/package/lens-4.13.1/docs/Control-Lens-Type.html#t:Lens-39-) suggesting the `P` suffix is an abbreviation of `Prime` because `'` isn't a valid character in a PureScript type name.

Perhaps an `_` suffix might be more appropriate.